### ISSUE TITLE
Docstrings on generators in a theory

### ISSUE THREAD

**Issue by olynch** [AlgebraicJulia/Gatlab.jl#78]
In Catlab, docstrings on the generators in a theory are saved to the theory structure. We should support this in Gatlab too.

**Comment by jpfairbanks** [comment]
@olynch did you know about dynamic docstrings?

https://docs.julialang.org/en/v1/manual/documentation/#Dynamic-documentation

You can have a method that dynamically constructs the docstring from values in the instance of a type. If there is ever a time for that, I think the docstring of a theory including a presentation of the theory is a perfect fit.

**Comment by olynch** [comment]
Waaaaaaat that would be so cool.

**Comment by jpfairbanks** [comment]
Yeah that also blew my mind when reading the docs on docs. I think that Documenter.jl will pull that into the docs live so our docs for the stdlib could just be an autogenerated HTML page with all the presentations typeset with latex.

It also avoids having to insert metaprogramming derived docstrings into the macroexpanded code.
